packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint-service.ts

```diff
@@ -151,6 +159,13 @@ export interface VpcEndpointServiceProps {
    */
   readonly acceptanceRequired?: boolean;
 
+  /**
+   * Indicates whether to enable the built-in Contributor Insights rules provided by AWS PrivateLink.
+   * @default false
+   *
+   */
+  readonly contributorInsightsEnabled?: boolean;
```

Comment:
```suggestion
  readonly contributorInsights?: boolean;
```

How about just `contributorInsights`?